nkms_eth/miner.py

```diff
@@ -0,0 +1,85 @@
+from typing import Tuple
+
+from .blockchain import Blockchain
+from .escrow import Escrow
+from .token import NuCypherKMSToken
+
+
+class Miner:
+    """Practically carrying a pickaxe"""
+
+    def __init__(self, blockchain: Blockchain, token: NuCypherKMSToken, escrow: Escrow, address=None):
+        self.blockchain = blockchain
+        self.token = token
+        self.address = address
+
+        self.escrow = escrow
+        if not escrow.contract:
+            raise Escrow.ContractDeploymentError('Escrow contract not deployed. Arm then deploy.')
```

Comment:
So you can't init a Miner until the Escrow contract is deployed?  That's worth noting in the `__init__` docstring.